### DISCUSSION TITLE
I was almost going crazy until I turned off Adblock Plus.

### DISCUSSION THREAD

**Discussion by igwen6w** [filamentphp/filament#10773] [General]
I found that one request has been constantly in a waiting state.

`/js/filament/tables/components/table.js?v=3.1.44.0`

![微信图片_20240112144029](https://github.com/filamentphp/filament/assets/24996622/3f7989de-b5c1-4016-861b-f50e0205cbe8)

![微信图片_20240112144051](https://github.com/filamentphp/filament/assets/24996622/4e0503fe-f702-4a64-9d8e-ffbf1a0402a1)

I spent several hours trying to figure out the reason until I turned off Adblock Plus. 
I don't know why it would pause the request for over 20 seconds, I don't understand the working principle of Adblock Plus, can anyone tell me?

I always thought that Adblock Plus would only block a certain request, but in this case, it just delayed it by over 20 seconds without blocking it. Why did it delay this request?

_My native language isn't English. The above content has been translated by ChatGpt. If there are inappropriate descriptions, please forgive me. This sentence is also translated._

**Comment by bernhardh** [comment]
Please have a look at this post: https://twitter.com/djgeisi/status/1747196179887137096

To sum it up: This is a bug in the extension and its already fixed in the next version of adblock.

**Reply by igwen6w** [reply]
Thank you. I also saw this tweet earlier
